src/components/transactions/StakingMigrate/StakingMigrateModalContent.tsx

```diff
@@ -85,15 +132,38 @@ export const StakingMigrateModalContent = () => {
         balanceText={<Trans>Wallet balance</Trans>}
       />
       <TxModalDetails gasLimit={gasLimit}>
-        <TxDetailsRow
```

Comment:
Relatively sure this is wrong.
This ui assumes perfect balance on in and out... which is never given.
https://app.balancer.fi/#/ethereum/pool/0x3de27efa2f1aa663ae5d458857e731c129069f29000200000000000000000588